app-frontend/src/app/components/colorCorrectPane/colorCorrectPane.html

```diff
@@ -5,16 +5,21 @@
       <a href class="btn btn-default btn-small" ng-click="$ctrl.$state.go('^.scenes')">Back</a>
     </div>
   </div>
-  <div ng-show="!$ctrl.errorLoadingHistogram" class="sidebar-static histogram">
-    <div class="histogram-loading-indicator" ng-show="$ctrl.loadingHistogram">
-      <i class="icon-load"></i>
+
+  <div class="sidebar-static histogram">
+    <div class="histogram-loading-indicator" ng-show="$ctrl.errorLoadingHistogram || $ctrl.loadingHistogram">
+      <!--Histogram is loading-->
+      <i ng-show="$ctrl.loadingHistogram" class="icon-load animated"></i>
+      <!--Histogram failed to load-->
+      <span ng-show="$ctrl.errorLoadingHistogram && !$ctrl.loadingHistogram">Error loading histogram</span>
```

Comment:
Spinner and error message are somehow showing at the same time even though that doesn't make any sense